tests/integration/request_test.py

```diff
@@ -100,3 +101,35 @@ def test_fuzzed_request(tag, id, mock_client):
     response = request.send()
 
     assert response.value == 'ok'
+
+
+@pytest.mark.parametrize(
+    'decorator_args, fuzzing_request_args',
+    [
+        (
+            {'tags': 'types'},
+            {'operation_id': 'get_expect_primitives', 'tag': 'types'},
+        ),
+        (
+            {},
```

Comment:
Might be worth adding a test that checks when you register a `tags` type, but try to fuzz a request that doesn't have the tag, it should not run the hook.

Otherwise, these two tests could pass, and the input for the first test case could just be a noop.

Reply:
Maybe also add a test for testing multiple registered post_fuzz hooks?

Reply:
Tests added